var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -64,8 +64,15 @@ class Libxsmm(MakefilePackage):
             description='With header-only installation')
     variant('generator', default=False,
             description='With generator executable(s)')
+    variant('blas', default='default', multi=False,
+            description='Control behavior of BLAS calls',
+            values=('default', '0', '1', '2'))
+    variant('large_jit_buffer', default=False,
+            description='Max. JIT buffer size increased to 256 KiB')
     conflicts('+header-only', when='@:1.6.2',
               msg='Header-only is available since v1.6.2!')
+    conflicts('+large_jit_buffer', when='@:1.17',
+              msg='large_jit_buffer is available since v1.17!')
```

Comment:
These conflicts can be removed and replaced with `when='@...'` in the variant itself. Also, these warning messages are unclear. Spack's version ranges are inclusive, so `conflicts(..., when='@:17')` implies that the option only works with 0.18+.

Reply:
Thanks @adamjstewart. Done.